docs/asyncapi.yaml

```diff
@@ -27,7 +27,7 @@ components:
       payload:
         oneOf:
           - $ref: '#/components/schemas/PaymentRequestSitiAgri'
-          - $ref: '#/components/schemas/PaymentRequestAHWR'
+          - $ref: '#/components/schemas/PaymentRequest'
```

Comment:
Why did you get rid of the AHWR part?

Reply:
Because it shouldn't be there.  AHWR is vet visits which doesn't come through this service, they go straight to enrichment. I suspect this was copy and pasted at some point.